packages/core/protocols/src/proto/dxos/client/services.proto

```diff
@@ -450,6 +451,17 @@ message Invitation {
 
   /// Path or identifier to navigate to after successful authentication.
   optional string target = 11;
+
+  // TODO(nf): should these exist elsewhere since they only have meaning to the host?
+  /// Host should resume invitation on startup until timeout.
+  optional bool persistent = 12;
+
+  optional google.protobuf.Timestamp created = 13;
+
+  /// Number of seconds the host will honor the invitation. 0 means the invitation persists forever.
+  // TODO(nf): some feedback mechanism or GC for immortal invitations?
+  // TODO(nf): should the creator have a way to make an invitation last as long as the client? this was the previous behavior.
+  optional int32 lifetime = 14;
```

Comment:
change to expiration ts?

Reply:
lol I had that originally, but I think Rich's comment swayed me to the benefits of seeing the TTL as a developer without having to calculate if we want creation as well anyway.

i don't have strong opinions either way, but I'm curious on your rationale?